.github/dependabot.yml

```diff
@@ -16,3 +16,7 @@ updates:
   labels:
     - "dependencies"
   open-pull-requests-limit: 10
+  ignore:
```

Comment:
Please add a comment containing `FIXME` to explain the the reason, and when to remove the ignorance.

Reply:
Done